lk2nd/device/dts/msm8953/msm8953-asus-zenfone3.dts

```diff
@@ -23,4 +23,32 @@
 			gpios = <&tlmm 13 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
 		};
 	};
+
+	panel {
+		compatible = "asus,zenfone3-panel", "lk2nd,panel";
+
+		qcom,mdss_dsi_ctc5p5_ili7807b_1080p_video {
+			compatible = "asus,ctc5p5_ili7807b";
+		};
+
+		qcom,mdss_dsi_tm5p5_r63350_1080p_video {
+			compatible = "asus,tm5p5_r63350";
+		};
+
+		qcom,mdss_dsi_tm5p2_r63350_1080p_video {
```

Comment:
Considering that they are two different phones (5.5 display and 5.2 display), are there any other differences or the display is the only one, and we can expect the hardware design to be the same? If not, I'd say we should probably split this into two nodes and use match-display here to separate 5.5 and 5.2...

Reply:
@TravMurav there are about seven variants for each screen size...
https://github.com/LineageOS/android_kernel_asus_msm8953/blob/d4ab72afd66672e7b3e49378e5eaf874c8e09440/include/linux/kernel.h#L22-L38

I don't really plan to port all the features of the Zenfone 3, as I only own the ZE520KL_MP. Should I create 14 nodes, or just 2 nodes (ze520kl and ze552kl)?

Reply:
Please only add those you can verify working

Reply:
ok I splitted them into two nodes based on the cmdline, wdyt?

Reply:
This looks good I think but please squash the commits